src/js/user-access.js

```diff
@@ -0,0 +1,21 @@
+import {products as getUserProducts} from 'next-session-client';
+import { SYNDICATION_ACCESS } from './config';
+export async function checkIfUserIsSyndicationCustomer () {
+
+	const response = await getUserProducts().catch(err => err);
```

Comment:
The `catch` part is not necessary since we are returning the same error

Reply:
done